test/test-pages/base-url-2/source.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <base href="https://base.url/"/>
```

Comment:
We should probably have tests with absolute and relative URLs with and without paths.